sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobAsyncClientBase.java

```diff
@@ -443,6 +447,7 @@ public String getVersionId() {
      * @return A {@link PollerFlux} that polls the blob copy operation until it has completed, has failed, or has been
      * cancelled.
      */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
```

Comment:
In the next version of azure-core there is a new return type of `LONG_RUNNING_OPERATION` that will be used for `PollerFlux` and `SyncPoller` APIs

Reply:
Another type that would be nice is something like a StreamResponse/Stream kind of thing. Right now I have to use Collection for that too and it doesnt make too much sense to me.